### PR TITLE
from Jen

### PR DESCRIPTION
**Pull request by jenneft**
Please review this code, let me know if we need to fix it?

### CODE REVIEW

**Review by cbanavi — CHANGES_REQUESTED** [review]
Add new comment

File: src/School/FeatureA.java [hunk 1]
```diff
@@ -0,0 +1,8 @@
+package School;
+
+public class FeatureA {
+    public static void main(String[] args) {
+        System.out.println("Hello  ");
```
Comment (jenneft) [inline src/School/FeatureA.java:5]: created main method with print statement "Hello"

**Review by jenneft — COMMENTED** [review]
Please check if this is acceptable

**Comment by cbanavi** [comment]
lgtm

**Review by cbanavi — APPROVED** [review]
lgtm

**Pull request merged**